bundles/org.openhab.binding.goveelan/src/main/java/org/openhab/binding/goveelan/internal/GoveeLanConfiguration.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.goveelan.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link GoveeLanConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Stefan Höhn - Initial contribution
+ */
+@NonNullByDefault
+public class GoveeLanConfiguration {
+    public static final String IPADDRESS = "hostname";
+    public static final String MAC_ADDRESS = "macaddress";
+    public static final String DEVICETYPE = "devicetype";
+    public static final String PRODUCTNAME = "productname";
+    public int refreshInterval = 3; // in seconds
```

Comment:
good catch. That happened during my testing because I wanted to check if it actually overridden by the default in the config.xml and I forgot to put it back ;-)

Reply:
Still 3 in README. 🙂

Reply:
I knew I had to change in two places but forget README ... sigh